scripts/amg/parser.ts

```diff
@@ -230,19 +230,19 @@ const runUpgrades = async () => {
   wb.worksheets.forEach(ws => {
     let faction = ws.getCell('A2').text.toLowerCase();
     if (faction === "separatist") {
-      faction = "Separatist Alliance";
+      faction = "separatistalliance";
     } else if (faction === "republic") {
-      faction = "Galactic Republic";
+      faction = "galacticrepublic";
     } else if (faction === "imperial" || faction.startsWith("upgrade points document")) {
-      faction = "Galactic Empire";
+      faction = "galacticempire";
     } else if (faction === "rebel") {
-      faction = "Rebel Alliance";
-    } else if (faction === "scum and villainy") {
-      faction = "Scum and Villainy";
+      faction = "rebelalliance";
+    } else if (faction === "scumandvillainy") {
+      faction = "scumandvillainy";
```

Comment:
Unclear if GitHub's mobile diff viewer is weird, but this looks like a no-op (will look on an actual computer later). Maybe just a find/replace victim?

Reply:
yeah, i think this was internally what was done to map the names to the factions. Should be safe to remove, I just didn't want to touch it in case something else depended on it and wasn't tested etc.

Reply:
Oh actually, these should probably not be changed, let me revert this file. I missed that this was the AMG parser

Reply:
looks like it should be checking if faction is "scum and villainy", must be lowercase cast at some point. Will update momentarily